projects/packages/jetpack-mu-wpcom/webpack.config.js

```diff
@@ -60,6 +61,9 @@ module.exports = [
 		plugins: [
 			...jetpackWebpackConfig.StandardPlugins( {
 				MiniCssExtractPlugin: { filename: '[name]/[name].css' },
+				DefinePlugin: {
+					__i18n_text_domain__: JSON.stringify( 'jetpack-mu-wpcom' ),
```

Comment:
Do we still need this one? If so, it would be better to just replace the `__i18n_text_domain__` with `jetpack-mu-wpcom` manually. What do you think?

Reply:
I think it is cleaner to remove it but it is used in the package `page-pattern-modal` that is published in npm. This package is only used in `jetpack-mu-wpcom` plugin for now. 

If we believe this package is only meant to be used here. I'll replace it in the package and publish a new version.

What do you think?

Reply:
Got it. We can add a comment here and address it later. If the package is only used by starter-page-templates, then we can consider moving it into jetpack as well in the future.

Reply:
Done